components/BrownBanner.tsx

```diff
@@ -12,7 +13,8 @@ export const BrownBanner: React.FC = () => {
           <Image src={OITLogo} alt="OIT Logo" width={200} height={0} priority />
         </a>
         <div className="ml-6 w-full flex flex-row justify-between items-center">
-          <h1 className="py-7 pl-6 h-full sm:text-xl lg:text-2xl font-semibold border-l border-black md:block hidden md:visible">
+          <h1 className="py-7 pl-6 h-full sm:text-xl lg:text-2xl font-semibold border-l border-black hidden md:flex items-center md:visible space-x-4">
```

Comment:
The space-x-4 is not working and I can't figure out why. Applying an inline style directly to the CBCLogo like this `style={{ marginRight: '1rem' }}` does work. Somehow the tailwind isn't being applied though.